index.html

```diff
@@ -4,11 +4,195 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap"
+      rel="stylesheet"
+    />
     <link rel="stylesheet" href="./styles/home/style.css" />
     <title>Hackathon</title>
   </head>
   <body>
-    <h1>Hackathon</h1>
-    <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Ex, itaque.</p>
+    <div id="fader"></div>
+    <header>
+      <div class="top-bar">
+        <ul class="top-bar-social">
+          <li>
+            <a href="#"
+              ><img src="./assets/icons/fabook.png" alt="facebook profile link"
+            /></a>
+          </li>
+          <li>
+            <a href="#"
+              ><img src="./assets/icons/twitter.png" alt="twitter profile link"
+            /></a>
+          </li>
+          <li>English</li>
+        </ul>
+      </div>
+      <nav>
+        <a href="#">
+          <img
+            class="desktop-element logo"
+            src="./assets/icons/logoipsum-280.svg"
+            alt="Hackathon logo"
+          />
+        </a>
+        <img
+          id="burger-icon"
+          class="mobile-element burger-icon"
+          src="./assets/icons/burger-icon.svg"
+          alt="mobile menu"
+        />
+        <ul id="nav-menu" class="nav-items">
+          <li>
+            <img
+              class="mobile-element close-btn"
+              src="./assets/icons/mob-menu-close-btn.svg"
+              alt="mobile menu close button"
+            />
+          </li>
+          <li class="mobile-element"><a href="#">Home</a></li>
+          <li><a href="./about.html">about</a></li>
+          <li><a href="#">program</a></li>
+          <li><a href="#">join</a></li>
+          <li><a href="#">sponser</a></li>
+          <li><a href="#">news</a></li>
+          <li><a class="nav-call-to-action" href="#">CC campaign</a></li>
+        </ul>
+      </nav>
```

Comment:
The pages should look almost identical to the original design as much as possible :+1: . Kindly check and make sure that the call to action link do not overlap and extends to the right as shown below. 

Your design             |  The actual design
:-------------------------:|:-------------------------:
![Screenshot from 2022-10-19 16-19-34](https://user-images.githubusercontent.com/65233860/196749026-3973a1c0-3e00-46bd-8936-a0d907416763.png)  |  ![Screenshot from 2022-10-19 16-21-10](https://user-images.githubusercontent.com/65233860/196749081-919a5299-ba9f-4449-aad0-011119fb1455.png)